src/libs/actions/Policy.js

```diff
@@ -190,7 +190,6 @@ function deletePolicy(policyID) {
             // Removing the workspace data from Onyx as well
             return Onyx.set(`${ONYXKEYS.COLLECTION.POLICY}${policyID}`, null);
         })
-        .then(() => Report.fetchAllReports(false))
```

Comment:
Looks like we added this so that report names for policy rooms would update when deleted -> https://github.com/Expensify/App/pull/7301

Since we are still using the `DeprecatedAPI.Policy_Delete()` here I wonder if we have plans to add this back in.

cc @chiragsalian since you got https://github.com/Expensify/Expensify/issues/215184

Reply:
Yeah I ended up adding a call to `AppInit::fetchReports($authToken);` on web (https://github.com/Expensify/Web-Expensify/pull/34172/files#diff-8367f109728fcc54db479ccc765a52d2115e52d35eba9c660e665be1a34411acR1613) to push the updated report names

Reply:
Ah got it, totally reviewed this PR before seeing that other one 🤦‍♂️